### PR TITLE
Callback to format the time

### PR DESCRIPTION
**Pull request by russ**
Adding a time_format callback to the configuration to allow the time to be formatted before being replaced in the template.

### CODE REVIEW

File: flowplayer.thumbnails.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
    requires:
    - Flowplayer HTML5 version 6.x or greater
-   revision: $GIT_ID$
+   revision: 49f9cc5
```
Comment (phloxic) [inline flowplayer.thumbnails.js]: Could you update without $GIT_ID$ expanded? Sorta defeats its purpose ;-) thx

**Comment by russ** [comment]
Sorry about that. I started off by copying the source of the file instead of forking.

File: README.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ flowplayer('#player', {
     title: 'Bauhaus',
     thumbnails: {
       template: 'thumbnails/bauhaus{time}.jpg'
+      time_format: function(t) {
+        return t + "-thumb.jpg";
```
Comment (phloxic) [inline README.md]: Could you give an example or use case where the function is _needed_?
The above could be simply set with `template: 'thumbnails/bauhaus{time}-thumb.jpg'`, couldn't it?

**Comment by russ** [comment]
I added a more detailed example. Please let me know if I can make it more clear. Thanks!

**Comment by phloxic** [comment]
@russ - thanks, even I get it now ;-)
Will take care of it later, hopefully tonight.

**Pull request merged**